tests/python/release_checklist/check_all_components_ui.py

```diff
@@ -0,0 +1,225 @@
+    "OutOfTreeTransform3DBatch": TestCase(
+        alternatives=[
+            rr.datatypes.TranslationRotationScale3D(
+                translation=(1, 2, 3), rotation=rr.datatypes.Quaternion(xyzw=[0, 0, 0, 1]), scale=(1, 1, 1)
+            ),
+            rr.datatypes.TranslationAndMat3x3(translation=(1, 2, 3), mat3x3=[[1, 0, 0], [0, 1, 0], [0, 0, 1]]),
+        ]
+    ),
+    "PinholeProjectionBatch": TestCase([(0, 1, 2), (3, 4, 5), (6, 7, 8)]),
+    "Position2DBatch": TestCase(batch=[(0, 1), (2, 3), (4, 5)]),
+    "Position3DBatch": TestCase(batch=[(0, 3, 4), (1, 4, 5), (2, 5, 6)]),
+    "RadiusBatch": TestCase(batch=[4.5, 5, 6, 7]),
+    "Range1DBatch": TestCase((0, 5)),
+    "Range2DBatch": TestCase(rr.datatypes.Range2D(x_range=(0, 5), y_range=(0, 5))),
+    "ResolutionBatch": TestCase((1920, 1080)),
+    "Rotation3DBatch": TestCase(
+        alternatives=[
+            rr.datatypes.Quaternion(xyzw=[0, 0, 0, 1]),
+            rr.datatypes.RotationAxisAngle(axis=(1, 0, 0), angle=rr.datatypes.Angle(rad=math.pi)),
+            rr.datatypes.RotationAxisAngle(axis=(1, 0, 0), angle=rr.datatypes.Angle(deg=180)),
+        ]
+    ),
+    "ScalarBatch": TestCase(3),
+    "ScalarScatteringBatch": TestCase(True),
+    "StrokeWidthBatch": TestCase(2.0),
+    "TensorDataBatch": TestCase(
+        alternatives=[
+            rr.datatypes.TensorData(array=np.random.randint(0, 255, (10, 10), dtype=np.uint8)),
+            rr.datatypes.TensorData(array=np.random.randint(0, 255, (10, 10, 3), dtype=np.uint8)),
+            rr.datatypes.TensorData(array=np.random.randint(0, 255, (5, 3, 6, 4), dtype=np.uint8)),
+        ]
+    ),
+    "Texcoord2DBatch": TestCase(batch=[(0, 0), (1, 1), (2, 2)]),
+    "TextBatch": TestCase("Hello world"),
+    "TextLogLevelBatch": TestCase(batch=["INFO", "CRITICAL", "WARNING"]),
+    "Transform3DBatch": TestCase(
+        alternatives=[
+            rr.datatypes.TranslationRotationScale3D(
+                translation=(1, 2, 3), rotation=rr.datatypes.Quaternion(xyzw=[0, 0, 0, 1]), scale=(1, 1, 1)
+            ),
+            rr.datatypes.TranslationAndMat3x3(translation=(1, 2, 3), mat3x3=[[1, 0, 0], [0, 1, 0], [0, 0, 1]]),
+        ]
+    ),
+    "TriangleIndicesBatch": TestCase(batch=[(0, 1, 2), (3, 4, 5), (6, 7, 8)]),
+    "Vector2DBatch": TestCase(batch=[(0, 1), (2, 3), (4, 5)]),
+    "Vector3DBatch": TestCase(batch=[(0, 3, 4), (1, 4, 5), (2, 5, 6)]),
+    "ViewCoordinatesBatch": TestCase(rr.components.ViewCoordinates.LBD),
+    "VisualizerOverridesBatch": TestCase(disabled=True),  # no Python-based serialization
+}
+
+
+def log_readme() -> None:
+    rr.log("readme", rr.TextDocument(README, media_type=rr.MediaType.MARKDOWN), static=True)
+
+
+def log_some_space_views() -> None:
+    # check that we didn't forget a component
+    assert set(ALL_COMPONENTS.keys()) == set([
+        c for c in dir(rr.components) if c.endswith("Batch")
+    ]), "The component list is incomplete"
```

Comment:
Please improve this error message to something more actionable ("Compinent $NAME missing in $FILEPATH" or similar)